tests/Eccube/Tests/Web/TopControllerTest.php

```diff
@@ -29,8 +29,43 @@ class TopControllerTest extends AbstractWebTestCase
 
     public function testRoutingIndex()
     {
-        $this->client->request('GET', $this->app['url_generator']->generate('homepage'));
+        $this->client->request('GET', $this->app->url('homepage'));
         $this->assertTrue($this->client->getResponse()->isSuccessful());
     }
 
+    /**
+     * testTopContent
+     */
+    public function testTopContent()
+    {
+        $client = $this->client;
+        $crawler = $client->request('GET', $this->app->url('homepage'));
```

Comment:
There are same error on all enviroment on travis-ci.
I think It's not good how to make crawlar.

you wrote:$this->app->url('homepage')
another test code:$this->app['url_generator']->generate('homepage')

test code is ok on your enviroment ?